In [7]:
# Consider the string given below and answer the following questions 
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"
print(len(string))


193


####  Calculate the number of characters in the string

-  193
-  200
-  300
-  500


In [2]:
# Type your code here

#### Count the number of occurences of 'a' in the string

-  1
-  2
-  10
-  5

In [9]:
# Type your code here
count = string.count('a')
print(count)


2


#### Identify whether the string starts with 'if'.

-  True
-  False

In [11]:
# Type your code here

print(string.startswith("Hello"))


False


#### Return the substring starting at index 63 and ending at index 88  [Both the indexes are inclusive]

-  'OQYWFClFhFGA'
-  'mCfgDYqbaBHAUvIWhMnvwZnEMV'
-  'IWYwGKpmZhnJiyzTslSIhSwv'
-  'JMEphzmifTkyqOMNpnOtXZ'

In [12]:
# Type your code here
string[63:89]


'mCfgDYqbaBHAUvIWhMnvwZnEMV'

#### Identify the character at index 45

-  a
-  b
-  z
-  x

In [16]:
# Type your code 
str1=string[45]
print(str1)

z


In [17]:
number='1353'
print(type(number))

<class 'str'>


In [18]:
number=34.721
print(float(int(number)))

34.0


In [19]:
number= alex
print(type(number))

NameError: name 'alex' is not defined

In [20]:
a = 8.9
b = 2
c = 5
print(a//b//c)

0.0


In [21]:
4 // 8 + ( 2.5 + 7.5 ) - 6 * 3


-8.0

In [22]:
5 % ( 2 + 12 ) ** 3 - 60 // ( 7 + 1 ) * 2


-9